### ISSUE TITLE
Runtime Training Error with 'Transformer' 

### ISSUE THREAD

**Issue by aastha19** [OpenNMT/OpenNMT-py#1676]
I wanted to train my model using the transformer architecture, and for that, I used the following command:

python train.py -data data/demo -save_model model_v43/demo-model --optim sgd --src_word_vec_size 8192 --tgt_word_vec_size 8192 --train_steps 200000 --enc_layers 2 --dec_layers 2 --dropout 0 -learning_rate 1  -gpu_rank 0 --encoder_type transformer --decoder_type transformer --heads 10

But this gives the following error:

![image](https://user-images.githubusercontent.com/14852345/70769783-e30d0980-1d90-11ea-97ff-3559858b287b.png)

Any suggestion as to how to handle this?

**Comment by francoishernandez** [comment]
See this post: http://forum.opennmt.net/t/transformer-error/3161
